src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukTabs.scala.html

```diff
@@ -18,11 +18,11 @@
 
 @this()
 
-@(params: Tabs)
+@(params: Tabs)(implicit messages: Messages)
 @import params._
 <div@id.mapNonEmpty {id => id="@id"} class="@toClasses("govuk-tabs", classes)"@toAttributes(attributes) data-module="govuk-tabs">
   <h2 class="govuk-tabs__title">
-    @HtmlFormat.escape(title)
+    @{title.map(HtmlFormat.escape _).getOrElse(messages("govuk.tabs.titleFallbackText"))}
```

Comment:
@JoPintoPaul I just realised something! The `@` will do HtmlFormat.escape for us - sorry, it's just if we want to skip escaping then we'd put `@Html(title)`